src/_data/mde.yml

```diff
@@ -38,83 +42,122 @@ extensions:
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       -
         name: 3.0.0
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       -
         name: 2.0.2
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       -
         name: 2.0.1
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       -
         name: 2.0.0
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       -
         name: 1.1.2
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
       - 
         name: 1.1.1
         support:
           2.3.0: general availability
           2.3.1: general availability
           2.3.2: general availability
           2.3.3: general availability
+          2.3.4: general availability
   -
     name: Magento Inventory
     versions:
       -
-        name: 1.6.1
+        name: 1.1.4
```

Comment:
This is correct?

Reply:
Yes, according to the canonical source for all VBEs and MDEs on the internal wiki.

Reply:
Confirmed that 1.6.1 doesn't exist. Also confirmed that later versions of Inventory usually work fine with earlier Magento versions and are considered "compatible" but not supported. I'll need to push some additional changes.